platform.md

```diff
@@ -269,7 +246,7 @@ or by executing `/cnb/lifecycle/creator`.
 > The meaning of any particular instance of the word **build** must be assessed in context
 
 #### Rebase
-When an updated run image with the same stack ID is available, an updated app image SHOULD be generated from the existing app image config by replacing the run image layers in the existing app image with the layers from the new run image.
+When an updated run image is available, an updated app image SHOULD be generated from the existing app image config by replacing the run image layers in the existing app image with the layers from the new run image.
```

Comment:
Do we care that the distribution name/versions match or is that covered by the "MUST" have ABI compatibility here? Another thought, should `io.buildpacks.id` match?